.github/workflows/ci.yml

```
@@ -0,0 +1,182 @@
+      env:
+        CGO_ENABLED: 0
+        GOOS: ${{ matrix.goos }}
+        GOARCH: ${{ matrix.goarch }}
+      run: |
+        VERSION=$(git describe --tags --always --dirty 2>/dev/null || echo "dev")
+        BUILD_TIME=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+        BINARY_NAME=framework-led-daemon
+        if [ "${{ matrix.goos }}" = "windows" ]; then
+          BINARY_NAME="${BINARY_NAME}.exe"
+        fi
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)